client/src/Components/Pages/Dashboard/ActiveJobs/style.css

```diff
@@ -0,0 +1,27 @@
+.find{
+  display: flex;
+  flex-direction: column;
+  background-color: #E0E0E0;
+ width: 300px;
+ border: 1px solid #011936;
+ border-radius: 10px;
+ padding: 25px;
+ margin: 25px;
+ font-family:Roboto;
```

Comment:
Do you think the whole app should use Roboto?  If so, you could add this font-family declaration to `body` in index.css and then you won't have to write it inside each component's css file.

Reply:
Done